packages/core/src/integrations/inboundfilters.ts

```diff
@@ -164,12 +164,14 @@ export class InboundFilters implements Integration {
       if (event.stacktrace) {
         // tslint:disable:no-unsafe-any
         const frames = (event as any).stacktrace.frames;
-        return frames[frames.length - 1].filename;
+	const {length: len, [len - 1]: last} = frames;
```

Comment:
here ugo @mitsuhiko @kamilogorek 

```suggestion
const {length: len, [len - 1]: {filename}} = frames;
```

Reply:
Amazing!

Reply:
We need more deconstruction and more shorthand notations.